spec/marsh_grass_spec.rb

```diff
@@ -93,27 +93,6 @@
     end
   end
 
-  context 'running tests for variable elapsed time' do
```

Comment:
Consider maybe marking these skipped instead of removing them. I'm not sure if we'll ever be able to replicate what Timecop was doing here, but I have used this feature once or twice.